package.json

```diff
@@ -3,16 +3,23 @@
   "version": "1.0.0",
   "description": "CS 362 Final Project",
   "scripts": {
+    "test": "jest",
```

Comment:
There's been changes on the main branch since you created this one, so you'll have to do a git pull origin main before we commit this one. This will change package.json and package-lock.json, and we'll want to keep the main branch's version of these. So, one easy way to do this is copy and paste the main branch's ones on top of these. BUT, you'll also need to re-install whatwg since it's not installed on main. Let me know if you need help and we can call!